### DISCUSSION TITLE
Downloading CoreNLP from Resource History

### DISCUSSION THREAD

**Discussion by NicolaHorst** [stanfordnlp/CoreNLP#1403] [General]
Hi, I just stumbled across a problem when trying to download the CoreNLP 3.9.2 version from the release history.
The problem is that the resource is marked as an insecure download because it was redirected via an insecure connection. As a result, the download cannot be started. 
There is no notification in Chrome indicating this situation, even though you open the development console. 
Perhaps you could include this in the installation instructions :)

**Comment by AngledLuffa** [comment]
If I switch it to https, does that help?  I did that with the first link,
the 4.5.5 link under "History"
